Makefile

```diff
@@ -47,8 +47,3 @@ doc-latexpdf:
 
 publish:
 	twine upload dist/*
-
-test:
-	coverage run --source=feedgen -m unittest discover -s tests
-	flake8 $$(find setup.py tests feedgen -name '*.py')
-	bandit -r feedgen
```

Comment:
Why are you deleting this?

Reply:
I want to follow the coding rules and fix the following problems.
```
flake8 $(find setup.py tests feedgen -name '*.py')
feedgen/ext/podcast_entry.py:100:1: W293 blank line contains whitespace
feedgen/feed.py:133:13: E741 ambiguous variable name 'l'
feedgen/entry.py:118:27: E741 ambiguous variable name 'l'
feedgen/entry.py:139:13: E741 ambiguous variable name 'l'
feedgen/entry.py:452:17: E741 ambiguous variable name 'l'
make: *** [test] Error 1
```

Reply:
I readded it into makefile (I remove it before just for testing)